Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Bugzilla55674.cs

```diff
@@ -0,0 +1,229 @@
+﻿using Xamarin.Forms.CustomAttributes;
+using Xamarin.Forms.Internals;
+using System.Linq;
+
+#if UITEST
+using Xamarin.UITest;
+using NUnit.Framework;
+#endif
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Preserve(AllMembers = true)]
+	[Issue(IssueTracker.Bugzilla, 55674, "[UWP] Using a navigation page changes the tabbed bar's color", PlatformAffected.UWP)]
+	public class Bugzilla55674 : TestTabbedPage
+	{
+		protected override void Init()
+		{
+			Children.Add(new FirstPage());
+			Children[0].Title = "Page One";
+
+			Children.Add(new SecondPage());
+			Children[1].Title = "Page Two";
+
+			Children.Add(new NavigationPage(new ThirdPage()));
+			Children[2].Title = "Page Three";
+
+			var tabbedPage = new TabbedPage();
+			tabbedPage.Children.Add(new ContentPage
+			{
+				Title = "Page One",
+				Content = new StackLayout
+				{
+					Children =
+					{
+						new Label { Text = "Page Four / TabbedPage"}
+					}
+				}
+			});
+			tabbedPage.Children.Add(new ContentPage
+			{
+				Title = "Page Two",
+				Content = new StackLayout
+				{
+					Children =
+					{
+						new Label { Text = "Page Four / TabbedPage's Page Two"}
+					}
+				}
+			});
+
+			Children.Add(new NavigationPage(tabbedPage));
```

Comment:
I believe I had force pushed this addition here if you get a chance to check it out.

Reply:
Excellent! While you're rebasing, can you put together some before/after screenshots for us? Thanks!!